lib/Sema/TypeCheckAvailability.cpp

```diff
@@ -2314,6 +2314,11 @@ class AvailabilityWalker : public ASTWalker {
         = TC.getFragileFunctionKind(DC);
   }
 
+  // FIXME: Remove this
+  bool shouldWalkAccessorsTheOldWay() override {
+    return true;
+  }
+
```

Comment:
I'm surprised at this one; why do we need to check availability at all on an accessor?  Or is this checking the function body?

Reply:
We visit accessors regardless of this method returning true or false. The difference is whether we visit them as children or peers of the storage. I need to tweak the few visitors that override this method to work with the new ordering, then I can remove the legacy code path altogether.

Reply:
Yeah, I understand the basics of why you've introduced this method, I just figured you'd seen actual problems at each of the sites where you did this.  Doesn't really matter.

Reply:
Yeah, there was a problem with availability checking of accessors that I haven't investigated yet. I don't expect it will be difficult to resolve.